graphql_compiler/query_formatting/common.py

```diff
@@ -47,8 +47,8 @@ def _deserialize_anonymous_json_argument(expected_type, value):
             GraphQLID: six.string_types
 
     Raises:
-        ValueError or arrow.parser.ParserMatchError if the value is not appropriate for the type.
-        These are the exceptions that the GraphQL parsers raise for invalid values.
+        ValueError if the value is not appropriate for the type. Value error is chosen because
```

Comment:
nit: `Value error` -> `ValueError`?

Reply:
Also I'm guessing this is true because the parser error raised by `arrow` is already a subclass of `ValueError`?